patterns/atoms/button/demo.css

```diff
@@ -1,16 +1,20 @@
 @import 'Pattern';
 
-table {
-	width: 100%;
-	border-collapse: collapse;
+.pp-demo {
+	display: flex;
+	padding: 20px;
 
-	td, th {
-		text-align: left;
-		padding: 8px 12px;
+	@nest &__container {
```

Comment:
Simple selectors can be [nest-prefixed](https://tabatkins.github.io/specs/css-nesting/#nest-prefixed) where @nest can be omitted